slack-lint/src/main/java/slack/lint/text/SpanPointMarkDangerousCheckDetector.kt

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2021 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package slack.lint.text
+
+import com.android.tools.lint.client.api.UElementHandler
+import com.android.tools.lint.detector.api.Category
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.JavaContext
+import com.android.tools.lint.detector.api.LintFix
+import com.android.tools.lint.detector.api.Severity
+import com.android.tools.lint.detector.api.SourceCodeScanner
+import com.android.tools.lint.detector.api.UastLintUtils
+import org.jetbrains.uast.UBinaryExpression
+import org.jetbrains.uast.UExpression
+import org.jetbrains.uast.UReferenceExpression
+import org.jetbrains.uast.UastBinaryOperator
+import org.jetbrains.uast.tryResolve
+import slack.lint.text.SpanPointMarkDangerousCheckDetector.Companion.ISSUE
+import slack.lint.util.resolveQualifiedNameOrNull
+import slack.lint.util.sourceImplementation
+
+/**
+ * Checks for SpanPointMarkDangerousCheck. See [ISSUE].
+ */
+class SpanPointMarkDangerousCheckDetector : Detector(), SourceCodeScanner {
```

Comment:
I've been thinking about this and I think "dangerous" is too strong of a word here. Let's pick something else, like "incorrect" or "missing mask check", something to that effect that's more descriptive

Reply:
How do you like `SpanMarkPointMissingMaskDetector`?

Reply:
sounds good to me 👍

Reply:
👍 Fixed in 961528cce768d48c0e762830fb62b80e92b89a13 and 34bb19ea23ab2d49f04e477f098fb43e91fae4ff